.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches:
+      - '**'
```

Comment:
An action will run on all pull requests if you simply omit the `branches:` clause.